src/interface/src/app/services/auth.service.ts

```diff
@@ -70,12 +70,16 @@ export class AuthService {
         email,
         first_name: firstName,
         last_name: lastName,
-      })
-      .pipe(
-        concatMap((_) => {
```

Comment:
Just double checking that this code was meant to be removed?

Reply:
That's correct. We're no longer logging in after sending the confirmation email.